ml/Host.cc

```diff
@@ -358,10 +362,14 @@ void DetectableHost::detectOnce() {
     double TotalTrainingDuration = 0.0;
     double MaxTrainingDuration = 0.0;
 
+    /*variable used to work out the percentage of the set anomalies over AnomalyBitCounterWindow*/
+    double AnomalyPercentage = 0.0;
```

Comment:
Move this to line 395. We have a checker that complains when variables are not declared/defined in the inner-most possible scope.

Reply:
done.